<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QMainWindow" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Депозитный калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма вклада</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок размещения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Налоговая ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Периодичность выплат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Список пополнений</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>391</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Список частичных 
          снятий</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>545</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>575</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма налога</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>605</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма на вкладе к концу срока</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>845</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>845</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>₽</string>
    </property>
   </widget>
   <widget class="QComboBox" name="periodDepositFormat">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>52</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>дней</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>135</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>% годовых</string>
    </property>
   </widget>
   <widget class="QComboBox" name="periodPayments">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>каждый день</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>каждую неделю</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в конце срока</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="percentCapitalization">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Капитализация процентов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsDepAdd">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>250</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInsDepPeriod">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>249</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>разовое</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>каждый день</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>каждую неделю</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в конце срока</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEditInsDep">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>249</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsDepClean">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>283</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isInsertDeposit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>249</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="isTakeDeposit">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>397</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Начало срока</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateStart">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCurrentDate">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>93</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая дата</string>
    </property>
   </widget>
   <widget class="QLabel" name="accruedAmount">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>545</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountTax">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>575</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountDepositEnd">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>605</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>637</y>
      <width>661</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_1">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>283</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="periodDeposit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="percentRate">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="amountDeposit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100000000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBoxInsDepAmount">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>249</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTakeDepClean">
    <property name="geometry">
     <rect>
      <x>568</x>
      <y>434</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>434</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditTakeDep">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>400</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTakeDepAdd">
    <property name="geometry">
     <rect>
      <x>567</x>
      <y>401</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTakeDepPeriod">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>400</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>разовое</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>каждый день</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>каждую неделю</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в конце срока</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBoxTakeDepAmount">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>400</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="tax">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>173</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isTax">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>170</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Учитывать налоги</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
